datafusion/substrait/tests/testdata/tpch_substrait_plans/query_1.json

```diff
@@ -1,810 +0,0 @@
-{
-    "extensionUris": [{
-      "extensionUriAnchor": 3,
-      "uri": "/functions_aggregate_generic.yaml"
-    }, {
-      "extensionUriAnchor": 2,
-      "uri": "/functions_arithmetic_decimal.yaml"
-    }, {
-      "extensionUriAnchor": 1,
-      "uri": "/functions_datetime.yaml"
-    }],
-    "extensions": [{
-      "extensionFunction": {
-        "extensionUriReference": 1,
-        "functionAnchor": 0,
-        "name": "lte:date_date"
-      }
-    }, {
-      "extensionFunction": {
-        "extensionUriReference": 1,
-        "functionAnchor": 1,
-        "name": "subtract:date_day"
-      }
-    }, {
-      "extensionFunction": {
-        "extensionUriReference": 2,
-        "functionAnchor": 2,
-        "name": "multiply:opt_decimal_decimal"
-      }
-    }, {
-      "extensionFunction": {
-        "extensionUriReference": 2,
-        "functionAnchor": 3,
-        "name": "subtract:opt_decimal_decimal"
-      }
-    }, {
-      "extensionFunction": {
-        "extensionUriReference": 2,
-        "functionAnchor": 4,
-        "name": "add:opt_decimal_decimal"
-      }
-    }, {
-      "extensionFunction": {
-        "extensionUriReference": 2,
-        "functionAnchor": 5,
-        "name": "sum:opt_decimal"
-      }
-    }, {
-      "extensionFunction": {
-        "extensionUriReference": 2,
-        "functionAnchor": 6,
-        "name": "avg:opt_decimal"
-      }
-    }, {
-      "extensionFunction": {
-        "extensionUriReference": 3,
-        "functionAnchor": 7,
-        "name": "count:opt"
-      }
-    }],
-    "relations": [{
-      "root": {
-        "input": {
-          "sort": {
-            "common": {
-              "direct": {
-              }
-            },
-            "input": {
-              "aggregate": {
-                "common": {
-                  "direct": {
-                  }
-                },
-                "input": {
-                  "project": {
-                    "common": {
-  
```

Comment:
The newly copied plans set the remap fields correctly. Their absence here didn't matter because DataFusion ignored them, but once I started adding remap support these plans broke because they were incorrect.